include/functions.h

```diff
@@ -380,21 +380,21 @@ u32 __osGetFpcCsr(void);
 // void __osPfsCheckRamArea(void);
 // void osPfsChecker(void);
 u32 osAiGetLength(void);
-s32 osEPiWriteIo(OSPiHandle* handle, u32 devAddr, u32 data);
+s32 osEPiWriteIo(OSPiHandle* handle, uintptr_t devAddr, u32 data);
 void osMapTLBRdb(void);
 void osYieldThread(void);
 void guTranslate(Mtx* mtx, f32 x, f32 y, f32 z);
 u32 __osGetCause(void);
 s32 __osContRamWrite(OSMesgQueue* mq, s32 channel, u16 address, u8* buffer, s32 force);
-s32 __osEPiRawWriteIo(OSPiHandle* handle, u32 devAddr, u32 data);
+s32 __osEPiRawWriteIo(OSPiHandle* handle, uintptr_t devAddr, u32 data);
 s32 osSetTimer(OSTimer* t, OSTime value, OSTime interval, OSMesgQueue* mq, OSMesg msg);
 void _Ldtob(_Pft* args, u8 type);
 // void _Ldunscale(void);
 void _Genld(_Pft* px, u8 code, u8* p, s16 nsig, s16 xexp);
 ldiv_t ldiv(long numer, long denom);
 lldiv_t lldiv(long long numer, long long denom);
 void _Litob(_Pft* args, u8 type);
-s32 __osSiRawWriteIo(u32 devAddr, u32 data);
+s32 __osSiRawWriteIo(uintptr_t devAddr, u32 data);
```

Comment:
There's not an awful lot of point in doing this in libultra IMO since we actually know the types used there (and we're intending to use a subrepo for it anyway as far as possible), but since you've done it it doesn't really matter.